src/app/pages/viewing/message-modal/message-modal.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="card shadow">
+  <div class="msg-header">
+    <div class="msg-header-group">
+      <table class="header-table">
+        <tr>
+          <td>
+            <span class="profile-pic-container-small mr-2">
+              <img class="profile-pic" src="{{ viewService.currentMessageModal.fromPic }}" alt="profile-pic">
+            </span>
+          </td>
+          <td>
+            <table>
+              <tr class="msg-header-text">
+                {{ viewService.currentMessageModal.fromName }}
+              </tr>
+              <tr class="msg-header-date">
+                {{ viewService.timeSince(viewService.nowMillis, 
+                viewService.currentMessageModal.timestamp.toMillis(),
+                viewService.currentMessageModal.timestamp.toDate().toDateString()) }}
+              </tr>
+            </table>
+          </td>
+        </tr>
+      </table>
+    </div>
+    <div class="card-header-group">
+      <table class="header-table">
+        <tr>
+          <td>
+            <button class="btn card-button" type="button"
+                    (click)="viewService.bookmarkMessage(viewService.currentMessageModal.id)">
+              <img class="card-icon" src="../../../../assets/images/icons/bookmark.svg" alt="bookmark">
+            </button>
+            <button type="button" class="btn" (click)="activeModal.close('Close click')">
+              <img class="card-icon" src="../../../../assets/images/icons/exit.svg" alt="close">
+            </button>
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
+  <div class="card-body message-card message" [ngStyle]="{'background': viewService.currentMessageModal.backColor, 'color': viewService.currentMessageModal.fontColor, 'font-family': viewService.fonts[viewService.currentMessageModal.fontFamily].fontFamily +','+ viewService.fonts[viewService.currentMessageModal.fontFamily].backupFont }">
```

Comment:
Please break this long line into multiple lines (Github formatting is weird so I couldnt suggest a change sry)